src/components/Interactive.js

```diff
@@ -46,7 +46,7 @@ const Interactive = Utils.forwardRefWithName('Interactive', ({
     className: `hover-style ${className}`,
     style: { ...style, ...cssVariables, cursor: computedCursor, outline },
     onKeyDown: onKeyDown || (e => {
-      if (e.key === 'Enter') {
+      if (e.key === 'Enter' || e.key === ' ') {
```

Comment:
This is a nice change. Looks like from what I could find on standard keyboard shortcuts this is better/more standard. Also previously space did a weird half scroll behavior in the workspaces list and this now gives it a nice defined behavior.

Reply:
Yep! In some widgets space and Enter are supposed to work differently (select boxes is an example) but for buttons they're supposed to do the same thing.

This change only applies when the user's focus in on an `Interactive` widget such as a `PrimaryButton`. So technically the space bar will still cause the page to scroll down a bit if the focus is on something else. But that is the standard expected browser behavior even if it seems a bit weird. :)